lib/screens/home_page/home_page.dart

```diff
@@ -15,50 +17,67 @@ class HomePage extends StatelessWidget {
-                InkWell(
-                    onTap: () {
-                      Navigator.pushReplacementNamed(context, '/login');
-                    },
-                    child: Icon(Icons.logout, color: Colors.white))
-              ],
-            ),
-            SizedBox(
-              height: size.height * 0.25,
-            ),
-            Center(
-              child: Column(
+        child: Container(
+          child: Column(
+            children: [
+              Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [Logo()],
+              ),
+              SizedBox(
+                height: size.height * 0.02,
+              ),
+              Row(
                 mainAxisAlignment: MainAxisAlignment.center,
                 children: [
-                  Text(text.second_screen,
-                      style: const TextStyle(fontSize: 20)),
-                  Padding(
-                    padding: const EdgeInsets.all(8.0),
-                    child: BlueButton(
-                        sizeWidth: size.width * 0.5,
-                        text: text.second_screen,
-                        onPressed: () => {}),
-                  )
+                  InkWell(
+                      onTap: () {
+                        Navigator.pushReplacementNamed(context, '/login');
+                      },
+                      child: Icon(Icons.logout, color: Colors.white))
                 ],
               ),
-            ),
-          ],
+              SizedBox(
+                height: size.height * 0.04,
+              ),
+              Container(
+                  width: size.width,
+                  height: size.height * 0.76,
+                  child: StreamBuilder(
+                      stream: todoBloc.todoListStream,
+                      builder:
+                          (context, AsyncSnapshot<List<TodoList>> snapshot) {
+                        var data = snapshot.data;
+
+                        if (!snapshot.hasData) {
+                          return Container(
+                            child: Center(child: Text(text.loading)),
+                          );
+                        }
+                        return SingleChildScrollView(
+                          child: ListView.builder(
+                              shrinkWrap: true,
+                              itemCount: data?.length,
+                              itemBuilder: (context, int index) {
+                                return ListTile(
+                                    leading: Icon(Icons.book),
+                                    title: Text(data![index].title),
+                                    subtitle: Text(data![index].description),
+                                    onTap: () {
+                                      /* react to the tile being tapped */
+                                    });
+                              }),
+                        );
+                      })),
```

Comment:
I think it's better to put commas between the parenthesis and the curly braces, this is to indent the code!